ray-operator/controllers/ray/common/service.go

```diff
@@ -307,9 +307,10 @@ func BuildHeadlessServiceForRayCluster(rayCluster rayv1.RayCluster) (*corev1.Ser
 			Labels:    labels,
 		},
 		Spec: corev1.ServiceSpec{
```

Comment:
For another PR, add unit tests for BuildHeadlessServiceForRayCluster

Reply:
I went ahead and added a test case in this PR ([c92042b](https://github.com/ray-project/kuberay/pull/2375/commits/c92042b0a86011f9c3f86193a30f3b78b374e596)) since it's a fairly small change.